src/main/java/com/worldwarofants/game/module/tutorial/TutorialController.java

```diff
@@ -1,17 +1,23 @@
 package com.worldwarofants.game.module.tutorial;
 
+import java.util.HashMap;
 import com.worldwarofants.game.arch.AbstractController;
 import com.worldwarofants.game.arch.module.IModuleNavigator;
 import com.worldwarofants.game.module.ModuleName;
+import com.worldwarofants.game.service.PlayerService;
 
 public class TutorialController extends AbstractController<TutorialView> {
 
-	public TutorialController(TutorialView view, IModuleNavigator navigator) {
+	public PlayerService playerService;
```

Comment:
In fact it could even be final too! Which is also another thing that you could start thinking about. If it can be final, then it might be good to make it final. Though that's slightly more situational.

Reply:
I have made it private and final.